userguide/snippets/es12.rst

```diff
@@ -62,8 +62,10 @@ Become Familiar With the ES12 System
 The ES12 has front panel indicators for power, locate ID, and fault.
 The fault indicator is on during the initial power-on self-test (POST)
 and turns off during normal operation. It turns on if the %brand%
-software issues an
-`alert <%docurl%/%brandlower%/tn_options.html#alert>`__.
+software issues an alert. See
+:menuselection:`Additional Options --> Alert`
```

Comment:
This is not a menu selection, but a location in the document.
"See the Alert section in the Additional Options section of the %brand%
`User Guide <...>`__."